onnxruntime/core/mlas/lib/sgemm.cpp

```diff
@@ -850,6 +850,33 @@ Return Value:
 
     }
 
+    if (N == 1 && ldb == 1 && ldc == 1 && alpha == 1.0f && (beta == 0.0f || beta == 1.0f))
+    {
+        // Both B and C are column-vectors that are contiguous in memory.
```

Comment:
Can you move the comment up above the if block to mirror the comment style from above (including // empty lines before/after)? This is old Windows kernel style...